### ISSUE TITLE
Directory is not set correctly in Win environment

### ISSUE THREAD

**Issue by wirfler** [nunomaduro/phpinsights#45]
If you try to run phpinsights in a Windows environment, it throws a Runtime Exception saying "The file `composer.json` must exist. You should run PHP Insights from the root of your project.". 

Reason: 
nunomaduro\phpinsights\src\Application\Console\Commands\AnalyseCommand.php:105

`if ($directory[0] !== DIRECTORY_SEPARATOR) {`

This check does obviously not work if $directoy is something like "C:\foldername\anotherfoldername", leading to a wrong concatenation in line 106.

**Comment by nunomaduro** [comment]
Can you fork and try to bug fix this? I don't have a windows machine.

**Comment by nunomaduro** [comment]
@wirfler Can you replace that line by: 

```php
        if ($directory[0] !== DIRECTORY_SEPARATOR && preg_match('~\A[A-Z]:(?![^/\\\\])~i', $directory) === 0) {
            $directory = (string) getcwd() . DIRECTORY_SEPARATOR . $directory;
        }
```

And check if it works?

**Comment by wirfler** [comment]
You were quicker. ;) 
Yes, works indeed!

**Comment by nunomaduro** [comment]
Fixed on the version `v1.0.2`. https://github.com/nunomaduro/phpinsights/commit/8bae26f096f6f9e39e3dc2e6c03ec4acb4e3f802.

Please make sure you perform: `composer update nunomaduro/phpinsights` to get the latest version.
